kotlinx-coroutines-core/jvm/test/sync/SemaphoreStressTest.kt

```diff
@@ -66,35 +59,48 @@ class SemaphoreStressTest : TestBase() {
         assertEquals(1, semaphore.availablePermits)
     }
 
+    /* This checks if repeated releases that race with cancellations put
```

Comment:
style: This should be KDoc commend formatted like a regual KDoc

Reply:
Ping! Either format this as KDoc:
```
/*
 * Text here (not leading star!)
 */
```
Or use multi-line commens in code (preferred for short comments):
```
// text
// more text
```

Reply:
I thought I have fixed this comment with the latest commit: https://github.com/Kotlin/kotlinx.coroutines/pull/1477/commits/63d563df22bb17d3121bdf5f0b830897c34b5a61

Do you mean that I should also replace the block comments in code with a group of single-line comments?

Reply:
You still have badly formatted comments in code like this one:
```
                /* Initially, we hold the permit and no one else can `acquire`,
                   otherwise it's a bug. */
```